### ISSUE TITLE
zoom-client automation

### ISSUE THREAD

**Issue by billwheatley** [billwheatley/provision-desktop#62]
zoom-client snap failed on Pop-OS 21.04, it has been commented out

```
fatal: [localhost]: FAILED! => {"changed": false, "channel": "stable", "classic": false, "cmd": "sh -c \"/usr/bin/snap install zoom-client\"", "msg": "Ooops! Snap installation failed while executing 'sh -c \"/usr/bin/snap install zoom-client\"', please examine logs and error output for more details.", "rc": 1, "stderr": "error: cannot perform the following tasks:\n- Run configure hook of \"zoom-client\" snap if present (run hook \"configure\": cp: cannot open '/var/lib/snapd/hostfs/etc/os-release' for reading: Permission denied)\n", "stderr_lines": ["error: cannot perform the following tasks:", "- Run configure hook of \"zoom-client\" snap if present (run hook \"configure\": cp: cannot open '/var/lib/snapd/hostfs/etc/os-release' for reading: Permission denied)"], "stdout": "", "stdout_lines": []}
```

Also may consider a role to install instead, there is no repo for this but download URL is predictable:
```
https://zoom.us/client/latest/zoom_x86_64.rpm
https://zoom.us/client/latest/zoom_amd64.deb
```

**Comment by billwheatley** [comment]
Note: zoom-client installed (at one point) on Pop 20.10 and directory `/var/lib/snapd/hostfs/etc/os-release` does not exist

**Comment by billwheatley** [comment]
Resolved by PR #70
